cmd/clusterctl/pkg/internal/test/reader.go

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Consider prefixing the file with `fake_`

Reply:
done